Model/Sales/Repository/PackageRepository.php

```diff
@@ -201,10 +201,10 @@ public function isAllProductsFitIn($products, string $packageType): bool
         if ($packageType === AbstractConsignment::PACKAGE_TYPE_MAILBOX_NAME) {
             $mailboxProcent = $this->getMailboxProcent();
             $mailboxProcent += ($this->getAttributesProductsOptions($products, 'fit_in_' . $packageType) * $products->getQty());
-            $mailboxWeight = $this->getConfigValue(self::XML_PATH_POSTNL_SETTINGS . 'mailbox/weight');
-            $orderWeight = $this->getWeight();
+            $mailboxWeight  = $this->getConfigValue(self::XML_PATH_POSTNL_SETTINGS . 'mailbox/weight');
```

Comment:
en hier hebben ze een spatie te veel :P

Reply:
misschien lijkt het zo op github, in mijn phpstorm ziet het er nettjes uit